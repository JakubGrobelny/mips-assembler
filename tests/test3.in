sll $3, $6, 23
srl $5, $3, 0x10
sllv $3, $5, $14 # random comment
mflo $5
# more comments
xori $3, $5, 0x1F
divu $4, $8