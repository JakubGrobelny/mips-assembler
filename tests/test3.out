00061DC0
00032C02
01C51804
00002812
38A3001F
0088001B